test/check-services

```diff
@@ -65,6 +65,10 @@ Description=Test Template for %I
 ExecStart=/usr/bin/test-service %I
 """)
 
+        # Only root can read the system.journal on RHEL 7.
+        if m.os == "rhel-7":
+            b.default_user = "root"
```

Comment:
Is there a bug about this? Do you know why there is a difference of behavior from Fedora?